c2corg_ui/static/js/utils.js

```diff
@@ -2,6 +2,26 @@ goog.provide('app.utils');
 
 
 /**
+ * @param {string} type Short document type code.
+ * @return {string} Full document type name.
+ */
+app.utils.getDoctype = function(type) {
+  // TODO add other types
+  switch (type) {
+    case 'w':
+      return 'waypoints';
+    case 'r':
+      return 'routes';
+    case 'o':
+      return 'outings';
+    case 'i':
+      return 'images';
+    default:
+      return '';
```

Comment:
Arf... there was a typo in the code example I copied from http://google.github.io/closure-library/api/namespace_goog_asserts.html :P 
Fixed! Thanks!